scripts/add-or-update-pr-issue-templates.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash 
+
+# This script will ensure a repo has the necessary pull request and issue templates as
+# part of our contributing guide: https://doc.gruntwork.io/guides/contributing/
+#
+# It creates or replaces the following files:
+#   .github/ISSUE_TEMPLATE/bug_report.md
+#   .github/ISSUE_TEMPLATE/feature_request.md
+#   .github/pull_request_template.md
+
+function create_bug_issue_template {
+cat << "EOF" > .github/ISSUE_TEMPLATE/bug_report.md
+---
+name: Bug report
+about: Create a bug report to help us improve.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+  Have any questions? Check out the contributing docs at https://docs.gruntwork.io/guides/contributing/, or
+  ask in this issue and a Gruntwork core maintainer will be happy to help :)
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior including the relevant Terraform/Terragrunt/Packer version number and any code snippets and module inputs you used.
+
+```hcl
+// paste code snippets here
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Nice to have**
+- [ ] Terminal output
+- [ ] Screenshots
+
+**Additional context**
+Add any other context about the problem here.
+EOF
+}
+
+function create_feature_issue_template {
+cat << "EOF" > .github/ISSUE_TEMPLATE/feature_request.md
+---
+name: Feature request
+about: Submit a feature request for this repo.
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!--
+  Have any questions? Check out the contributing docs at https://docs.gruntwork.io/guides/contributing/, or
+  ask in this issue and a Gruntwork core maintainer will be happy to help :)
+-->
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
+EOF
+}
+
+function create_pr_template {
+cat << "EOF" > .github/pull_request_template.md
```

Comment:
NIT: should these be `.md` files checked into this `git-xargs` repo? You could then use `cp` to copy them to the proper destination. It would then make it easier to read / edit / preview those files than when they are inlined within a Bash script.

Reply:
@brikis98 we can, but it would be the first script example that copies external files to a repo. I guess this is not such a bad thing, but it will also make the script slightly less portable. I.e. The user would need to ensure they copy a folder containing the script and all of the required templates for it to work 100% correctly. We could fail hard if one of the templates is missing. @zackproser what are your thoughts?

Reply:
Moving the markdown files to the `git-xargs` repo would make them slightly easier to read, but we'd also have the extra step of ensuring we've updated and pulled on the repo each time for running the latest script, as Rob points out. I don't have a strong opinion here, so I'd say go with whatever is easier for you. 

Makes me think `git-xargs` might do well with a means of fingerprinting the script or command it ran each time and adding that info to a PR description or something - for easier coordination in the common workflow of needing to make several small updates to a script following the first couple of runs. This would make it easier for colleagues to know which version of a script generated which PRs.

Reply:
> Makes me think git-xargs might do well with a means of fingerprinting the script or command it ran each time and adding that info to a PR description or something - for easier coordination in the common workflow of needing to make several small updates to a script following the first couple of runs. This would make it easier for colleagues to know which version of a script generated which PRs.

Yeah, that's a good idea. We've been in a very similar territory recently planning an internal project. It appears RenovateBot actually stores a fair bit of state using pull requests. i.e: It knows whether a repo has been onboarded or not based on the status of a PR. I think it would be handy if `git-xargs` could by default sprinkle some metadata into the PRs.